flake.nix

```diff
@@ -1,75 +1,115 @@
 {
-  description =
-    "ripgrep, but also search in PDFs, E-Books, Office documents, zip, tar.gz, etc.";
+  description = "ripgrep, but also search in PDFs, E-Books, Office documents, zip, tar.gz, etc.";
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs";
-
     crane = {
       url = "github:ipetkov/crane";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-
-    flake-utils.url = "github:numtide/flake-utils";
-
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    flake-root.url = "github:srid/flake-root";
```

Comment:
not really an expert in nix but it seems a bit weird to pull in some random code just to make this package (which is not very special) work

Reply:
That's fair. IMO the couple extra inputs are worth it to make the flake more featureful and easier to maintain, but they're definitely not mandatory. I've pushed 81d04fb8e91fa277ed0b7a7dca8e7a53d0da2c5f without those additional inputs and the corresponding refactor.